source/Google/MobileAds/Enums.cs

```diff
@@ -38,51 +38,57 @@ public enum AdLoaderAdType
 		UnifiedNative
 	}
 
-	//GADRequest file
 	[Native]
-	public enum Gender : long
+	public enum AdValuePrecision : long
 	{
-		Unknown,
-		Male,
-		Female
+		Unknown = 0,
+		Estimated = 1,
+		PublisherProvided = 2,
+		Precise = 3
 	}
 
-	//GADRequestError file
 	[Native]
-	public enum ErrorCode : long
+	public enum PresentationErrorCode : long
 	{
-		InvalidRequest,
-		NoFill,
-		NetworkError,
-		ServerError,
-		OSVersionTooLow,
-		Timeout,
-		InterstitialAlreadyUsed,
-		MediationDataError,
-		MediationAdapterError,
-		MediationNoFill,
-		MediationInvalidAdSize,
-		InternalError,
-		InvalidArgument,
-		ReceivedInvalidResponse,
-		RewardedAdAlreadyUsed
+		AdNotReady = 15,
+		AdTooLarge = 16,
+		Internal = 17,
+		AdAlreadyUsed = 18
 	}
 
+	//GADRequest file
 	[Native]
-	public enum SearchBorderType : ulong
+	public enum Gender : long
 	{
-		None,
-		Dashed,
-		Dotted,
-		Solid
+		Unknown,
+		Male,
+		Female
 	}
 
+	//GADRequestError file
 	[Native]
-	public enum SearchCallButtonColor : ulong
+	public enum ErrorCode : long
 	{
-		Light,
-		Medium,
-		Dark
+		InvalidRequest = 0,
+		NoFill = 1,
+		NetworkError = 2,
+		ServerError = 3,
+		OSVersionTooLow = 4,
+		Timeout = 5,
+		[Obsolete ("Use AdAlreadyUsed enum value instead.")]
+		InterstitialAlreadyUsed = 6,
+		MediationDataError = 7,
+		MediationAdapterError = 8,
+		MediationInvalidAdSize = 10,
+		InternalError = 11,
+		InvalidArgument = 12,
+		ReceivedInvalidResponse = 13,
+		[Obsolete("Use AdAlreadyUsed enum value instead.")]
+		RewardedAdAlreadyUsed = 14,
```

Comment:
You can go the extra mile and do this instead 
```suggestion
		RewardedAdAlreadyUsed = AdAlreadyUsed,
```

Same for the other obsoletes